spec/lib/vanagon/platform/dsl_spec.rb

```diff
@@ -58,33 +66,31 @@
       expect(SecureRandom).to receive(:hex).and_return(hex_value)
       plat.instance_eval(el_5_platform_block)
       plat.yum_repo(el_definition)
-      expect(plat._platform.provisioning).to include("curl -o '/etc/yum.repos.d/#{hex_value}-pl-puppet-agent-0.2.1-el-7-x86_64.repo' '#{el_definition}'")
+      expect(plat._platform.provisioning[0]).to include('rpm -q curl', 'yum -y install curl')
+      expect(plat._platform.provisioning[1]).to include(
+        'curl', '--silent', '--show-error', '--fail',
+        "-o '/etc/yum.repos.d/#{hex_value}-pl-puppet-agent-0.2.1-el-7-x86_64.repo' '#{el_definition}'")
     end
 
     it "works for specifically redhat platforms" do
       plat = Vanagon::Platform::DSL.new('redhat-7-fixture')
       expect(SecureRandom).to receive(:hex).and_return(hex_value)
       plat.instance_eval(redhat_7_platform_block)
       plat.yum_repo(el_definition)
-      expect(plat._platform.provisioning).to include("curl -o '/etc/yum.repos.d/#{hex_value}-pl-puppet-agent-0.2.1-el-7-x86_64.repo' '#{el_definition}'")
-    end
-
-    # This test currently covers wrlinux 5 and 7
```

Comment:
Can the removal of this test go in a separate commit? It seems pretty unrelated to the curl work.

Reply:
Restored the code in this branch.

Reply:
I didn't mean that you should put it back, just make the change in a separate commit. but this is fine too.

Reply:
Reverting it was the better choice. I looked further and a correct, complete removal of this platform, even just from the rspec files, would involve more changes. I'll file another ticket to do that.